releases/nightly-build/compose-files/docker-compose-nexus-redis-no-secty-arm64.yml

```diff
@@ -226,15 +226,22 @@ services:
       - data
 
   rulesengine:
-    image: nexus3.edgexfoundry.org:10004/docker-support-rulesengine-arm64:master
+    image: emqx/kuiper:0.3.0
     ports:
-      - "48075:48075"
-    container_name: edgex-support-rulesengine
-    hostname: edgex-support-rulesengine
+      - "9081:9081"
+      - "20498:20498"
+    container_name: edgex-kuiper
+    hostname: edgex-kuiper
     networks:
       edgex-network:
         aliases:
-          - edgex-support-rulesengine
+          - edgex-kuiper
+    environment:
+      EDGEX_SERVER: edgex-core-data
+      EDGEX_SERVICE_SERVER: http://edgex-core-data:48080
+      EDGEX_TOPIC: events
+      EDGEX_PROTOCOL: tcp
+      EDGEX_PORT: 5563
```

Comment:
Port needs to be `5566` so you receive events from app-service-rules that may be doing some filtering.

Also need to add environment to app-service-rules to set the Publish topic to be `events` as it is currently blank. 
Added the following for app-service-rules environment:
```
Binding_PublishTopic: events
```

Reply:
Thanks for your advice, I have updated the code